src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -434,8 +490,8 @@ ApiGlobe.prototype.orbit = function orbit(value) {
  */
 
 ApiGlobe.prototype.setLayerOpacity = function setLayerOpacity(id, opacity) {
-    this.scene.getMap().setLayerOpacity(id, opacity);
-    this.scene.renderScene3D();
+    this.scene.layersConfiguration.setLayerAttribute(id, 'opacity', opacity);
+    this.scene.notifyChange(0, true);
```

Comment:
No real reason. `Scene` should probably expose only 1 method to update its content/rendering.